Styles.css

```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-size: 2rem;
+    color: #ffffff;
+    background-image: url("BG.jpeg");
+    background-position: center;
+    background-attachment: fixed;
+    background-size: 100vw auto;
+    background-repeat: no-repeat;
+    font-family: "Helvetica", Arial, sans-serif;
+}
+
+@media only screen and (max-aspect-ratio: 4016/6016) {
+    body {
+        background-size: auto 100vh;
+    }
+}
+
+a {
+    text-decoration: inherit;
+    color: inherit;
+}
+
+header {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    -ms-flex-align: start;
+    align-items: flex-start;
+    padding: 20px;
+    height: 20vh;
+    min-height: 140px;
+}
+
+.Homebutton {
+    font-family: "Brush Script MT", Arial, sans-serif;
+    font-size: 3rem;
+    width: 7rem;
```

Comment:
this is an inline element, so width does nothing.
also dont use rems for anything but font sizes. u dont want widths to change based on the users operating system